src/pages/billing/List.js

```diff
@@ -192,30 +177,49 @@ export const BillingList = _.flow(
     this.state = {
       billingProjects: null,
       creatingBillingProject: false,
+      billingAccounts: null,
       ...StateHistory.get()
     }
   }
 
-  async componentDidMount() {
-    this.loadProjects()
-  }
+  componentDidMount = Utils.withBusyState(isBusy => this.setState({ isBusy }),
+    () => Promise.all([
```

Comment:
Does this need the `Promise.all`? From the last time I did something like this, I remember this stuff being fiddly. However I'm a little suspicious about the bundling of these into a single promise that is not `await`ed.

Reply:
What were the issues? Promise.all is used throughout the code base. When I don't wait on both requests, the billing account will just "pop in" to the select, which yields a slightly jarring UX. On the await, wrapping this function in withBusyState adds an async await to the function. I can add an additional await for better clarity if needed